app/graphql/types/organizational_unit_type.rb

```diff
@@ -11,4 +11,18 @@
   field :displayName, types.String do
     description 'Display name of the organizational unit'
   end
+
+  field :repositories, !types[Types::RepositoryType] do
+    description 'List of repositories owned by this organizational unit'
+
+    argument :limit, types.Int do
+      description 'Maximum number of repositories to list'
+      default_value 20
+    end
+
+    argument :skip, types.Int do
+      description 'Skip the first n repositories'
+      default_value 0
+    end
```

Comment:
Just checked. It's possible. I think we can even remove the complete field definition.